reddit_liveupdate/public/static/css/liveupdate.less

```diff
@@ -591,42 +619,39 @@ aside.sidebar {
         }
     }
 
-    #liveupdate-options {
-        float: none;
-        clear: both;
-        margin-top: 1em;
-        text-align: right;
-        width: auto;
-    }
-
     aside.sidebar {
         float: none;
         margin: 0 0 2em 0;
-        max-width: 700px;
         width: auto;
         opacity: 1;
         transition: opacity 0;
 
-        section {
-            border: 1px solid #ccc;
-            background-color: #fafafa;
-            padding: 10px;
-            font-size: 13px;
+        border: 1px solid #ccc;
+        background-color: #fafafa;
+        padding: 10px;
+        font-size: 13px;
```

Comment:
actually, i'm 95% sure this can just be removed.  the `.sidebar` element also has the `.md-container` class, which sets `font-size: small;` (equivalent to 13px)